velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -807,6 +807,55 @@ struct MakeYMIntervalFunction {
   }
 };
 
+template <typename T>
+struct MakeDTIntervalFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(out_type<IntervalDayTime>& result) {
```

Comment:
Spark provides default values for the unprovided parameters, so we can only register the four-param implementation.